src/navigations/root-stack.js

```diff
@@ -287,27 +310,27 @@ const AuthenticatedNavigator = () => {
 
         <AuthenticatedStack.Screen
           name="ProfileReplyComment"
-          component={withSafeAreaView(withKeyboardWrapper(ProfileReplyComment))}
+          component={withKeyboardWrapper(ProfileReplyComment)}
           options={{headerShown: false}}
         />
         <AuthenticatedStack.Screen
           name="OtherProfileReplyComment"
-          component={withSafeAreaView(withKeyboardWrapper(OtherProfileReplyComment))}
+          component={withKeyboardWrapper(OtherProfileReplyComment)}
           options={{headerShown: false}}
         />
         <AuthenticatedStack.Screen
           name="PostDetailPage"
-          component={withSafeAreaView(withKeyboardWrapper(PostDetailPage))}
+          component={withKeyboardWrapper(PostDetailPage)}
           options={{headerShown: false}}
         />
         <AuthenticatedStack.Screen
           name="ProfilePostDetailPage"
-          component={withSafeAreaView(withKeyboardWrapper(ProfilePostDetail))}
+          component={withKeyboardWrapper(ProfilePostDetail)}
           options={{headerShown: false}}
         />
         <AuthenticatedStack.Screen
           name="OtherProfilePostDetailPage"
-          component={withSafeAreaView(withKeyboardWrapper(OtherProfilePostDetail))}
+          component={withKeyboardWrapper(OtherProfilePostDetail)}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `withSafeAreaView` wrapper has been removed from several components (`ProfileReplyComment`, `OtherProfileReplyComment`, `PostDetailPage`, `ProfilePostDetail`, `OtherProfilePostDetail`). Ensure that this does not affect the layout of these screens on devices with notches or rounded corners (like iPhone X and above).



<!-- This is an auto-generated comment by OSS CodeRabbit -->